src/fsipy/simulations/offset_stenosis.py

```diff
@@ -64,20 +64,20 @@ def set_problem_parameters(default_variables, **namespace):
         lambda_s=lambda_s_val,  # Solid Young's modulus [Pa]
         dx_s_id=2,  # ID of marker in the solid domain
         # FSI parameters
-        fsi_region=[0.003, 0, 0, 0.01],  # x, y, and z coordinate of FSI region center, and radius of FSI region sphere
+        fsi_x_range=[0.0, 0.016],  # range of x coordinates for fsi region
         # Simulation parameters
         folder="offset_stenosis_results",  # Folder name generated for the simulation
         mesh_path="mesh/file_stenosis.h5",
-        FC_file="FC_MCA_10",  # File name containing the fourier coefficients for the flow waveform
-        P_FC_File="FC_Pressure",  # File name containing the fourier coefficients for the pressure waveform
+        FC_file="FC_MCA_10",  # File name containing the Fourier coefficients for the flow waveform
```

Comment:
There is no file `FC_MCA_10`. Should this be `FC_file="MCA_10"` or should `MCA_10` be renamed to `FC_MCA_10`?

Reply:
`FC_MCA_10` is the Fourier coefficient version of the `MCA_10`. Not sure if we need to keep both of them, but I think we can discuss this later with others.